src/scss/components/_thumbnail.scss

```diff
@@ -17,84 +17,50 @@
 
 .thumbnail {
   width: 100%;
+  height: 100%;
   position: relative;
   overflow: hidden;
   border-radius: $borderRadius;
   transition: transform .2s linear;
   will-change: transform;
 
-  &__title {
-    color: get-color(rastaGray);
-  }
+  &--action {
+    display: flex;
+    flex-direction: column-reverse;
 
-  &__description {
-    color: get-color(blackNeutral, lighter);
+    &:hover {
+      transition: transform .2s linear;
```

Comment:
@bsoares @iondrimba talking with Bernardo, our designer, he told me that will be a refactor of all animations on version 2 of Gaiden (they have your own control version nomenclature). Nice tip about easinsgs.net! I normally use animate.css for this.

I don't know if this is a block to PR for now, because I just move this code from a location to here. Maybe a animations refactor PR makes more sense.